namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-query-production/resources/rds.tf

```diff
@@ -0,0 +1,51 @@
+################################################################################
+# Track a Query (Correspondence Tool Staff)
+# Application RDS (PostgreSQL)
+#################################################################################
+
+module "track_a_query_rds" {
+  source                     = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.12"
+  cluster_name               = var.cluster_name
+  cluster_state_bucket       = var.cluster_state_bucket
+  team_name                  = "correspondence"
+  business-unit              = "Central Digital"
+  application                = "track-a-query"
+  is-production              = "true"
+  namespace                  = var.namespace
+  db_engine                  = "postgres"
+  db_engine_version          = "12.3"
```

Comment:
hi - please just set this to "12", as a new minor release 12.5 is already available
If you do not want to allow minor upgrades, use release https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/releases/tag/5.13.1 and set `allow_major_version_upgrade` false

Reply:
@razvan-moj  thank you for reviewing this,   just for my knowledge,  if I put 
`db_engine_version          = "12"` ,   the AWS platform will create a instance of 12.5 by default? 

for the flag of `allow_major_version_upgrade`,  I copied this value from our other environments like staging, development, qa, do I need to change them too?

Reply:
@razvan-moj , done the update, could you have a another look? thank you!